src/test/java/org/cactoos/list/JoinedTest.java

```diff
@@ -129,35 +139,83 @@ public void containsAll() throws Exception {
         );
     }
 
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     public void addAll() throws Exception {
-        new Joined<String>(
-            new ListOf<String>("addAll")
-        ).addAll(new ListOf<>("new addAll"));
+        final List<String> joined = new Joined<String>(
+            new ListOf<>("One"),
+            new ListOf<>("Two")
+        );
+        joined.addAll(
+            new ListOf<>("Three")
+        );
+        new Assertion<>(
+            "must be able to addAll elements specified",
+            joined.get(2),
+            new IsEqual<>("Three")
+        ).affirm();
     }
 
-    @Test(expected = UnsupportedOperationException.class)
-    public void addAllSecond() throws Exception {
-        new Joined<String>(
-            new ListOf<String>("addAll1")
-        ).addAll(22, new ListOf<>("new addAll1"));
+    @Test
+    public void addAllInFront() throws Exception {
+        final List<String> joined = new Joined<String>(
```

Comment:
@fanifieiev Here also: better all the elements to see if they are the correct order